src/components/GenerateGifForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import './GenerateGifForm.css';
+
+class GenerateGifForm extends Component {
+  constructor(props) {
+    super(props);
+    this.handleInputUpdate = this.handleInputUpdate.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(evt) {
+    evt.preventDefault();
+    this.props.handleGenerateGIF();
+    this.props.updateText('');
+    this.props.updateTextColor('');
+  }
+
+  handleInputUpdate(evt) {
+    if (evt.target.name === 'caption') {
```

Comment:
This doesn't feel very DRY to me - @alissarenz, thoughts on refactoring?

Reply:
Better to use a switch statement.

Reply:
Done